README.md

```diff
@@ -1,17 +1,100 @@
 # Conda Recipes
 
-Example recipes for the new conda build system.  Use
+Community-maintained repository of conda package build recipes
+for 592+ conda packages.
 
-    conda build <recipe directory>
+* Package maintainers SHOULD upgrade packages
+  when appropriate.
 
-You have to use conda >= 1.7
-[conda](https://github.com/continuumio/conda).
+* Conda recipes MAY run other scripts:
 
-See http://docs.continuum.io/conda/build.html for information on how to make a recipe,
-or just look at the examples here.
 
-This project (`conda-recipes`) is in the public domain.   Note that this statement
+  * at build time by specifying ``script:`` in ``meta.yaml``
+    (default: ``build.bat`` / ``build.sh``)
+  * at build time in `build.bat` (Windows) and `build.sh` (OSX, Linux)
+  * before the package is installed in ``pre-link.sh``
+  * after the package is installed in ``post-link.sh``
+  * before the package is removed in ``pre-unlink.sh``
```

Comment:
But only if you set `script:`. I'd say they are required, at least in the big picture of conda-build